### PR TITLE
Fix: Styling of the resetPassword page

### PR DESCRIPTION
**Pull request by fandridis**
The original reset password page has a nearly invisible title and styling different than the rest of auth pages.
Changed the styling to look like the other pages.

### CODE REVIEW

File: src/js/modules/core/components/ResetPasswordPage.tsx [hunk 1]
```diff
@@ -8,15 +8,22 @@ import { resetPassword, clearEmailState } from "../coreActions";
...
 
 const styles = (theme: Theme): object => ({
   ResetPasswordPage: {
     display: "flex",
     flexDirection: "column",
     alignItems: "center",
-    padding: "5%",
-    width: "100vw",
-    backgroundColor: theme.formBackground
+    // padding: "5%",
```
Comment (NicholasLYang) [inline src/js/modules/core/components/ResetPasswordPage.tsx]: Let's remove this

File: src/js/modules/core/components/ResetPasswordPage.tsx [hunk 2]
```diff
@@ -8,15 +8,22 @@ import { resetPassword, clearEmailState } from "../coreActions";
...
+    width: "100%",
+    maxWidth: "28em",
```
Comment (NicholasLYang) [inline src/js/modules/core/components/ResetPasswordPage.tsx]: Could we use `theme.containerSmallWidth`?

File: src/js/modules/core/components/ResetPasswordPage.tsx [hunk 3]
```diff
@@ -8,15 +8,22 @@ import { resetPassword, clearEmailState } from "../coreActions";
...
+    minWidth: "500px",
+    backgroundColor: theme.formBackground,
+    color: "#202020",
```
Comment (NicholasLYang) [inline src/js/modules/core/components/ResetPasswordPage.tsx]: Let's use `theme.secondFont`

**Comment by fandridis** [comment]
I made the changes. Took me a minute to understand that **theme.secondFont** is actually referring to color :D (fontColor and secondFontColor might have been more appropriate names)

Great work so far! I wish I was able to participate to the hackaton.

**Pull request merged**